references/classification/presets.py

```diff
@@ -68,6 +68,9 @@ def __init__(
         if random_erase_prob > 0:
             transforms.append(T.RandomErasing(p=random_erase_prob))
 
+        if use_v2:
```

Comment:
technically, we do not need this as we do not wrap to `Image` in the classification pipeline, right ?

Reply:
removed